docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+database:
+  image: "sameersbn/mysql:latest"
+  environment:
+    DB_NAME: staytus
+    DB_USER: staytus
+    DB_PASS: staytus
+  volumes:
+    - "/opt/docker/staytus-database:/var/lib/mysql"
+staytus:
+  #image: "adamcooke/staytus:latest"
+  image: "staytus-test"
```

Comment:
Is this correct? I think the commented out name is what we should be using for end-users - this requires you've got a staytus-test image already available locally.